devices/tuya.js

```diff
@@ -4279,4 +4279,208 @@ module.exports = [
         toZigbee: [],
         configure: tuya.configureMagicPacket,
     },
+    {
+        fingerprint: [{
+            modelID: 'TS0001',
+            manufacturerName: '_TZ3000_bmqxalil',
+        }],
+        model: 'TS0001_switch_1_gang',
+        vendor: 'TuYa',
+        description: '1-Gang switch with backlight',
+        fromZigbee: [tuya.fz.backlight_mode_off_on, fz.on_off, tuya.fz.power_on_behavior_2],
+        toZigbee: [tz.on_off, tuya.tz.power_on_behavior_2, tuya.tz.backlight_indicator_mode_2],
+        exposes: [
+            e.switch(),
+            e.power_on_behavior(),
+            exposes.binary('backlight_mode', ea.ALL, 'on', 'off').withDescription('Backlight Mode'),
+        ],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            await tuya.configureMagicPacket(device, coordinatorEndpoint, logger);
+            await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+        },
+    },
+    {
+        fingerprint: [{
+            modelID: 'TS0002',
+            manufacturerName: '_TZ3000_in5qxhtt',
+        }],
+        model: 'TS0002_switch_2_gang',
+        vendor: 'TuYa',
+        description: '2-Gang switch with backlight',
+        fromZigbee: [tuya.fz.backlight_mode_off_on, fz.on_off, tuya.fz.power_on_behavior_2],
+        toZigbee: [tz.on_off, tuya.tz.power_on_behavior_2, tuya.tz.backlight_indicator_mode_2],
+        exposes: [
+            e.switch().withEndpoint('l1'),
+            e.switch().withEndpoint('l2'),
+            exposes.binary('backlight_mode', ea.ALL, 'on', 'off').withDescription('Backlight Mode'),
```

Comment:
I see that `power_on_behavior` expose is missing here, was that intentional?

Reply:
fixed it latest, didn't push it yet, will explain at the bottom